weekly-report/main.py

```diff
@@ -115,20 +121,21 @@ def handler(event=None, context=None):
             <h3>Introduction</h3>
             <p>This report provides an overview of the recent seismic activity globally, with a particular focus on USA. It includes details about the magnitude, location, depth, and potential impact of detected earthquakes. The data is sourced from the United States Geological Survey (USGS).</p>
             <h3>Recent Seismic Activity</h3>
-            <p>From June 1, 2024,to June 24, 2024, the seismic monitoring network recorded a total of 15 significant earthquakes. The following visual summarizes the key details of these events.</p>
+            <p>From June 1, 2024,to {CURRENT_DATE}, the seismic monitoring network recorded a total of 15 significant earthquakes. The following visual summarizes the key details of these events.</p>
             <img style="width: 500; height: 300" src="{mag_html}">
             <h3>What's happening in USA?</h3>
             <img style="width: 250; height: 150" src="{state_html}">
         </body>
     """
     file_data = convert_html_to_pdf(html_report)
+    logging.info(f"weekly report converted from html to PDF")
     prefix = get_prefix(CURRENT_DATE)
     file_name = f"{CURRENT_DATE}.pdf"
     upload_historical_readings(
         s_client, ENV['STORAGE_BUCKET_NAME'], prefix, file_name, file_data)
-
-    return {"success": "complete"}
+    logging.info(f"Weekly PDF report uploaded to {ENV['STORAGE_BUCKET_NAME']}")
 
 
 if __name__ == "__main__":
+    logging.basicConfig(level=logging.INFO)
```

Comment:
I like the use of logging to create messages that can be viewed on the log system in aws